Code/src/main/java/nl/utwente/group10/ui/components/lines/Connection.java

```diff
@@ -31,13 +37,13 @@ public Connection() {
     }
 
     public Connection(OutputAnchor from) {
-        this.setStartAnchor(from);
-        setEndPosition(from.getCenterInPane());
+        tryAddAnchor(from);
+        setEndPosition(getScenePoint(from));
```

Comment:
Nee, het is een regressie. `getScenePoint` is verwijderd (want niet meer nodig) in master.